server/database/queries/updateStudent.js

```diff
@@ -0,0 +1,12 @@
+const connection = require('../config/db_connection');
+
+const putStudent = (userInfo, id) => {
+  const queryValues = Object.keys(userInfo).map(info => userInfo[info]);
```

Comment:
you can't use this method if you didn't make validation because it relies on the order of  keys and that will cause a error if the front end sends a body in an incorrect order like this object : 
```
{
 "password": "rami", 
 "mobile_phone": "ahmed",
 "lastname": "ahmed",
 "home_phone": "059198",
 "address": "gaza",
 "level": "5",
 "firstname": "hassan"
}
```
@Fatmasiam  same issues btw